src/main/java/com/wolApi/wolApi/BibleVerseController.java

```diff
@@ -1,26 +1,35 @@
 package com.wolApi.wolApi;
 
-import org.jsoup.Connection;
-import org.jsoup.HttpStatusException;
 import org.jsoup.Jsoup;
 import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PathVariable;
 
 import java.io.IOException;
+import java.util.LinkedList;
 
 
 @RestController
 @RequestMapping("/api/v1/bible-verses")
 public class BibleVerseController {
-    @GetMapping("/get-verse")
-    public Verse getVerse() throws IOException {
-        Document doc = Jsoup.connect("https://wol.jw.org/en/wol/b/r1/lp-e/nwtsty/1/1#study=discover&v=1:1:1")
-                .get();
-        doc.select("p").forEach(System.out::println);
+    @GetMapping("/get-verse/{book}/{chapter}")
```

Comment:
should probably change the get mapping to "get-verses"

Reply:
Couldn't we do a:

For getting a whole chapter:
`/get-verse/{book}/{chapter}`

For getting a single verse:
`/get-verse/{book}/{chapter}/{verseNumber}`

Root API routes are usually singular

Reply:
Yeah we could do it that way... just want to make it as clear as possible what each route does. I guess as long as it relates to the resource then it makes sense.